atsign-foundation/atsign.dev-3.0
# Added content to pricing/costs as well as some minor fixes

**- What I did**
-Added pricing/costs page
-Fixed Small typo
**- How I did it**
-Wrote content for the pricing/costs
- Deleted Repeated Sentence.
**- How to verify it**
- See privacy.md
- See pricing_and_costs.md
**- Description for the changelog**
